content/docs/reference/certificates.mdx

```diff
@@ -152,52 +151,10 @@ Kubernetes does not support `certificate_authority`
 </TabItem>
 </Tabs>
 
-## Client Certificate Authority {#client-certificate-authority}
-
 :::caution
 
-This setting is deprecated; it has been moved to a new **Downstream mTLS Settings** group (see the [**Certificate Authority**](/docs/reference/downstream-mtls-settings#ca) setting there).
+We've removed support for the `client_ca` and `client_ca_file` settings. To configure client certificates for downstream connections, see the **Downstream mTLS Settings** group (see the [**Certificate Authority**](/docs/reference/downstream-mtls-settings#ca) setting there).
 
-This setting will be treated as an alias for the new setting, but will be removed in a future Pomerium release.
+If you need the older `client_ca` and `client_ca_file` settings, please see the [**v0.25.0 Client Certificate Authority**](https://0-25-0.docs.pomerium.com/docs/reference/certificates#client-certificate-authority) documentation, or an earlier Pomerium version.
```

Comment:
Again, the setting has moved, but otherwise should behave just the same as in previous versions. I'm not sure it's helpful to include this link, especially if the `#client-certificate-authority` anchor no longer links here.